<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChessboardCoverageClass</class>
 <widget class="QMainWindow" name="ChessboardCoverageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>963</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChessboardCoverage</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
    <item>
     <widget class="QGroupBox" name="gBox_chessBoard">
      <property name="title">
       <string>棋盘</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidget"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gBox_chessBoardSet">
      <property name="title">
       <string>棋盘相关操作</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>棋盘设置</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,1">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>大小因子（指数k）：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="sBox_sizeK"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>x:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>特殊块位置：</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="sBox_x"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>y:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSpinBox" name="sBox_y"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="pbtn_resetChessboard">
             <property name="text">
              <string>重设棋盘</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbtn_CoverageOperate">
             <property name="text">
              <string>覆盖加载</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2"/>
         </item>
         <item>
          <widget class="QGroupBox" name="gBox_display">
           <property name="title">
            <string>展示</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="3" column="0" colspan="2">
             <widget class="QPushButton" name="pbtn_autoDisplay">
              <property name="text">
               <string>自动展示覆盖过程</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pbtn_nextStep">
              <property name="text">
               <string>下一步</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="pbtn_skipToInitial">
              <property name="text">
               <string>回到初始状态</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pbtn_lastStep">
              <property name="text">
               <string>上一步</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pbtn_skipToFinal">
              <property name="text">
               <string>跳到终止状态</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="label_loadInfo">
              <property name="text">
               <string>还未加载，加载后可以操作！！！！</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>308</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ChessboardCoverage.qrc"/>
 </resources>
 <connections/>
</ui>
